.github/workflows/pr.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+on:
+  pull_request: {}
```

Comment:
f9fd90af5ad8233ccab54baf8a61a35e9134d311 says "we will configure CI to only run check_release on PRs to master". Do we have to configure this here?

Reply:
We do! I will fix that, thank you.